### ISSUE TITLE
Resolve issues and update a11y tests

### ISSUE THREAD

**Issue by josmar-crwdstffng** [angular/angular#45608]
### Description

# Resolve issues and update a11y tests

**Background**:

To enable the following PRs to merge, the a11y score threshold was reduced from `100` to `98`.

*   [#45325](https://github.com/angular/angular/pull/45325)
*   [#45590](https://github.com/angular/angular/pull/45590)

**Affected files**:

*   `aio/scripts/test-aio-a11y.mjs`
     *   Line 28: `  'cli': 98,`
*   `api/core/Directive`
*   `cli`
*   `guide/docs-style-guide`
*   `start/start-routing`
*   `tutorial`
 
**Requests**:
 
1.  Update the a11y tests to provide test details when score is less than `100`.
1.  Enable verbose mode for a11y tests to display full log details.

**Requirements**:

1.  Updated tests to display failing conditions.
1.  Update tests to display possible solutions matching from MDN or Lighthouse output.
1.  Updated threshold scores set to `100` for all `MIN_SCORES_PER_PAGE` values.

### What is the affected URL?

angular.io/cli

### Please provide the expected behavior vs the actual behavior you encountered

The a11y test to provide the details on where the content is causing the score to be less than `100`.

**Comment by josmar-crwdstffng** [comment]
Please assign any documentation updates to @josmar-crwdstffng and @mgechev.
